dotcom-rendering/scripts/test/build-check.js

```diff
@@ -41,17 +41,11 @@ const fileExists = async (glob) => {
 	}
 
 	[
-		'sentryLoader.js',
```

Comment:
Is just a driveby fix for the single bundle stuff?

Reply:
Oop! I should have added this to the PR description, sorry! Yes, this reflects the [single entry point](https://github.com/guardian/dotcom-rendering/pull/7086) change. This [test was failing silently](https://github.com/guardian/dotcom-rendering/actions/runs/4135033898/jobs/7146945837#step:6:12) for some time!